### ISSUE TITLE
Honeybadger not defined on the client-side

### ISSUE THREAD

**Issue by nickytonline** [forem/forem#11032]
**Describe the bug**

PR #11030 is a hot fix for sorting out a race condition where the Honeybadger client is not yet defined. It is a temporary fix that needs a better solution.

**To Reproduce**

1. Remove the temporary fix in #11030 
2. Load any page and see the error `Honeybadger is not defined` in the browser console.

**Expected behavior**

**Screenshots**

![image](https://user-images.githubusercontent.com/833231/96949343-af713480-14b5-11eb-8d37-afc28b360630.png)

**Desktop (please complete the following information):**

- OS, version: N/A
- Browser, version: N/A

**Smartphone (please complete the following information):**

- Device: N/A
- OS, version: N/A
- Browser, version: N/A

**Additional context**

**Comment by rhymes** [comment]
@nickytonline do you mean the temporary fix in https://github.com/forem/forem/pull/11030 ?

**Comment by nickytonline** [comment]
Yes, not sure where that other number came from. 🙃 I've updated the PR description.

**Comment by mikeydorje** [comment]
Closing as stale. The temporary fix with `setTimeout` is still in place but no permanent solution has been implemented in 5+ years.

If this issue affects your instance, please reopen with details.
